### ISSUE TITLE
Full day event renders over two days in month view

### ISSUE THREAD

**Issue by mrminning** [MMM-CalendarExt2/MMM-CalendarExt2#84]
I use Google Calendar to create events to be shown in MMM-CalendarExt2 month view. I have some reoccuring full day events, and all of them renders over the correct day and the following day.

This is a snippet of my config.js
`
{
      module: "MMM-CalendarExt2",
      config: {
        calendars: [
          {
            forceLocalTZ: false,
            beforeDays: 0,
            afterDays: 60,
            icon: "mdi-castle",
            url: ""
          },
        ],
        views: [
          {
            mode: "week", 
            slotCount: 3,
            locale: "sv-SE",
            position: "middle_center",
          },
        ],
        scenes: [
          {
            name: "DEFAULT",
          },
        ],  
      }
    },
`

Here is an example of failing VEVENT:
`
BEGIN:VEVENT
DTSTART;VALUE=DATE:20200602
DTEND;VALUE=DATE:20200603
RRULE:FREQ=WEEKLY;WKST=MO;INTERVAL=2;BYDAY=TU
DTSTAMP:20200603T175106Z
UID:secret@google.com
CREATED:20200603T174826Z
DESCRIPTION:Städning
LAST-MODIFIED:20200603T174826Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Städning
TRANSP:TRANSPARENT
BEGIN:VALARM
ACTION:DISPLAY
DESCRIPTION:This is an event reminder
TRIGGER:-P0DT0H30M0S
END:VALARM
END:VEVENT
`
I've seen that the code rendered for the event shows like this
`< div class="event fullday targetday oneday recurred starthere endhere" data-calendar-name="0" data-calendar-seq="0"
     data-start-date="1592265600" data-end-date="1592352000" data-duration="86400" data-title="Städning"
     data-location="" style="left: 422.7px; width: 406.217px;">
    < div class="eventMain">
        < svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true"
             width="1em" height="1em"
             style="vertical-align: -0.125em;-ms-transform: rotate(360deg); -webkit-transform: rotate(360deg); transform: rotate(360deg);"
             preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24" class="iconify eventIcon" data-icon="mdi-castle">
            < path d="M2 13h2v2h2v-2h2v2h2v-2h2v2h2v-5l3-3V1h2l4 2l-4 2v2l3 3v12H11v-3a2 2 0 0 0-2-2a2 2 0 0 0-2 2v3H2v-9m16-3c-.55 0-1 .54-1 1.2V13h2v-1.8c0-.66-.45-1.2-1-1.2z"
                  fill="currentColor">< /path>
        < /svg>
        < div class="eventTitle">Städning< /div>
        < div class="eventTime">
            < div class="startDate start date">6/16< /div>
            < div class="startTime start time">02:00< /div>
            < div class="startDateTime start dateTime">6/16 02:00< /div>
            < div class="endDate end date">6/17< /div>
            < div class="endTime end time">02:00< /div>
            < div class="endDateTime end dateTime">6/17 02:00< /div>
        < /div>
    < /div>
    < div class="eventSub">
        < div class="eventDescription">Städning< /div>
        < div class="eventLocation">< /div>
    < /div>
< /div>
`
I found this which might help find the issue:

- data-start-date="1592265600" data-end-date="1592352000" is correct if you don't consider any timezones.
- data-duration="86400" is one day.
- <div class="eventTime"/> seems to have added 2h. Probably because I'm in DST (European Daylight Savings Time).
- I have tried forceLocalTZ: true in my config.js without any luck. 

The beginning of my ical-file starts like this:
`
BEGIN:VCALENDAR
PRODID:-//Google Inc//Google Calendar 70.9054//EN
VERSION:2.0
CALSCALE:GREGORIAN
METHOD:PUBLISH
X-WR-CALNAME:secret@gmail.com
X-WR-TIMEZONE:Europe/Stockholm
`
[files.zip](https://github.com/MMM-CalendarExt2/MMM-CalendarExt2/files/4738249/files.zip)

I can't get html and/or javascript to render ok in the comment, so I added the code in a zip file.

**Comment by mrminning** [comment]
I found the solution. It was in the docker container the problem was. https://github.com/bastilimbach/docker-MagicMirror/issues/15

**Comment by klaernie** [comment]
@mrminning Cool that you found the solution. Was already scratching my head about this one ;)

**Comment by bigrjsuto** [comment]
I'm experiencing the same issue. All of the full-day / all-day events are displaying over the intended day and the day after. I am running MM on a Debian LXC so I'm not sure if it's the same issue. I started experiencing it a several weeks/months ago (I'm not sure, finally got annoyed enough to start trying to fix). 

I added "en-US" to the config  for this module in the config.js and it didn't do anything. I'm not getting any errors in the MM logs, either. 

I don't know how to pull the render code like mrminning did, so figuring that out may help me.
